<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Vstup:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Výstup:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="codeButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Šifrovat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decodeButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dešifrovat</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>127</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="aKey"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>127</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>B:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bKey"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
